lib/datasource/ruby-version/index.ts

```diff
@@ -1,25 +1,27 @@
-const { parse } = require('node-html-parser');
-const { logger } = require('../../logger');
+import { parse } from 'node-html-parser';
+import { logger } from '../../logger';
 
-const got = require('../../util/got');
-const { isVersion } = require('../../versioning/ruby');
-
-module.exports = {
-  getPkgReleases,
-};
+import got from '../../util/got';
+import { isVersion } from '../../versioning/ruby';
+import { PkgReleaseConfig, ReleaseResult } from '../common';
 
 const rubyVersionsUrl = 'https://www.ruby-lang.org/en/downloads/releases/';
 
-async function getPkgReleases() {
+export async function getPkgReleases(
+  _config?: PkgReleaseConfig
+): Promise<ReleaseResult> {
   // First check the persistent cache
   const cacheNamespace = 'datasource-ruby-version';
-  const cachedResult = await renovateCache.get(cacheNamespace, 'all');
+  const cachedResult = await renovateCache.get<ReleaseResult>(
```

Comment:
This is not `Promise<ReleaseResult>`, or doesn't matter eitherway?

Reply:
seems ok for ts / js